domain/src/test/java/org/oppia/android/domain/classify/rules/dragAndDropSortInput/DragDropSortInputHasElementXAtPositionYRuleClassifierProviderTest.kt

```diff
@@ -26,11 +24,21 @@ import kotlin.test.fail
 @LooperMode(LooperMode.Mode.PAUSED)
 @Config(manifest = Config.NONE)
 class DragDropSortInputHasElementXAtPositionYRuleClassifierProviderTest {
-  private val NON_NEGATIVE_VALUE_0 = createNonNegativeInt(value = 1)
-  private val NON_NEGATIVE_VALUE_1 = createNonNegativeInt(value = 2)
-  private val STRING_VALUE_2 = createString(value = "test item 2")
-  private val STRING_VALUE_3 = createString(value = "test item invalid")
-  private val LIST_OF_SETS_OF_HTML_STRING_VALUE = createListOfSetsOfHtmlStrings()
+
+  private val NON_NEGATIVE_VALUE_TEST_1 =
+    InteractionObjectTestBuilder.createNonNegativeInt(value = 1)
+
+  private val NON_NEGATIVE_VALUE_TEST_2 =
+    InteractionObjectTestBuilder.createNonNegativeInt(value = 2)
+
+  private val STRING_VALUE_TEST_1 =
```

Comment:
this should not be `STRING_VALUE_TEST_1`. It should be something related to the test it is going to use for.